tests/integration/StatisticsLayoutTest.php

```diff
@@ -0,0 +1,235 @@
+            /** @var XiboLibrary $media */
+            $flag = true;
+            foreach ($this->startMedias as $startMedia) {
+                if ($startMedia->mediaId == $media->mediaId) {
+                    $flag = false;
+                }
+            }
+            if ($flag) {
+                try {
+                    $media->deleteAssigned();
+                } catch (\Exception $e) {
+                    $this->getLogger()->error('Media: Unable to delete ' . $media->mediaId . '. E:' . $e->getMessage());
+                }
+            }
+        }
+
+        // tearDown all layouts that weren't there initially
+        $finalLayouts = (new XiboLayout($this->getEntityProvider()))->get(['start' => 0, 'length' => 10000]);
+        // Loop over any remaining layouts and nuke them
+        foreach ($finalLayouts as $layout) {
+            /** @var XiboLayout $layout */
+            $flag = true;
+            foreach ($this->startLayouts as $startLayout) {
+                if ($startLayout->layoutId == $layout->layoutId) {
+                    $flag = false;
+                }
+            }
+            if ($flag) {
+                try {
+                    $layout->delete();
+                } catch (\Exception $e) {
+                    $this->getLogger()->error('Layout: Unable to delete ' . $layout->layoutId . '. E:' . $e->getMessage());
+                }
+            }
+        }
+
+        // Tear down any displays that weren't there before
+        $finalDisplays = (new XiboDisplay($this->getEntityProvider()))->get(['start' => 0, 'length' => 10000]);
+
+        // Loop over any remaining displays and nuke them
+        foreach ($finalDisplays as $display) {
+            /** @var XiboDisplay $display */
+            $flag = true;
+            foreach ($this->startDisplays as $startDisplay) {
+                if ($startDisplay->displayId == $display->displayId) {
+                    $flag = false;
+                }
+            }
+            if ($flag) {
+                try {
+                    $display->delete();
+                } catch (\Exception $e) {
+                    $this->getLogger()->error('Display: Unable to delete ' . $display->displayId . '. E:' . $e->getMessage());
+                }
+            }
+        }
+        parent::tearDown();
+    }
+
+    /**
+     * Test the method call with default values
+     * @group broken
+     */
+    public function testListAll()
+    {
+        $this->client->get('/stats');
+
+        $this->assertSame(200, $this->client->response->status());
+        $this->assertNotEmpty($this->client->response->body());
+        $object = json_decode($this->client->response->body());
+        $this->assertObjectHasAttribute('data', $object, $this->client->response->body());
+    }
+
+    /**
+     * Check if proof of play statistics are correct
+     */
+    public function testProof()
+    {
+        // Create a Display
+        $display = $this->createDisplay();
```

Comment:
We are creating a specific display and layout in this test - could we have these items in `setup` instead? and then in `teardown` could we just call delete on these objects?

If we only add specific items in setup and then delete them in teardown, i think we should be able to remove all of the other setup/teardown code?